__tests__/index-test.js

```diff
@@ -87,6 +87,75 @@ export default componentName;
   expect(code).toMatchSnapshot();
 });
 
+it('arrow-noreturn-annotate-fragment snapshot matches', () => {
+  const { code } = babel.transform(
+`import React, { Component, Fragment } from 'react';
+
+const componentName = () => (
+  <Fragment>
+    <h1>Hello world</h1>
+  </Fragment>
+);
+
+export default componentName;
+`,
+    {
+      filename: "./filename-test.js",
+      presets: ["@babel/preset-react"],
+      plugins: [
+        [plugin, { "annotate-fragments": true }]
+      ]
+    },
+  );
+  expect(code).toMatchSnapshot();
+});
+
+it('arrow-noreturn-annotate-fragment-once snapshot matches', () => {
+  const { code } = babel.transform(
+`import React, { Component, Fragment } from 'react';
+
+const componentName = () => (
+  <Fragment>
+    <h1>Hello world</h1>
+    <h1>Hola Sol</h1>
+  </Fragment>
+);
+
+export default componentName;
+`,
+    {
+      filename: "./filename-test.js",
+      presets: ["@babel/preset-react"],
+      plugins: [
+        [plugin, { "annotate-fragments": true }]
+      ]
+    },
+  );
+  expect(code).toMatchSnapshot();
+});
+
+
+it('arrow-noreturn-annotate-fragment-no-whitespace snapshot matches', () => {
+  const { code } = babel.transform(
+`import React, { Component, Fragment } from 'react';
+
+const componentName = () => (
+  <Fragment><h1>Hello world</h1><h1>Hola Sol</h1></Fragment>
+);
+
+export default componentName;
+`,
+    {
+      filename: "./filename-test.js",
+      presets: ["@babel/preset-react"],
+      plugins: [
+        [plugin, { "annotate-fragments": true }]
+      ]
+    },
+  );
+  expect(code).toMatchSnapshot();
+});
+
```

Comment:
Would it be a good idea to test the expected behavior of a `Fragment` that contains no nested Components like `<Fragment>Hello world</Fragment>` ?